package.json

```
@@ -96,6 +102,7 @@
     "path-browserify": "^1.0.1",
     "prettier": "^2.4.1",
     "prettier-eslint": "15.0.1",
+    "prismjs": "^1.19.0",
```

Actually, prismjs still has a newer version in `package-lock.json`:

https://github.com/mdn/bob/blob/01e3f7473688b6cc036aa826041be0f17791d715/package-lock.json#L19851-L19856

Afaik a workaround is to `npm remove prismjs; npm install prismjs@1.19.0` and add the caret back manually only afterwards.

```suggestion
    "prismjs": "1.19.0",
```